content/tracing/advanced_usage/_index.md

```diff
@@ -279,6 +279,88 @@ if (($span = GlobalTracer::get()->getActiveSpan()) !== null){
 {{% /tab %}}
 {{< /tabs >}}
 
+## Change Agent Hostname
+
+Configure your application level tracers to submit traces to a custom Agent hostname. See the examples below for each supported language:
+
+{{< tabs >}}
+{{% tab "Java" %}}
+
+The Java Tracing Module automatically looks for and initializes with the ENV variables DD_AGENT_HOST and DD_TRACE_AGENT_PORT:
+
+```bash
+java -javaagent:/path/to/the/dd-java-agent.jar -jar /your/app.jar
+```
+
+You can also use system properties:
+
+```bash
+java -javaagent:/path/to/the/dd-java-agent.jar \
+     -Ddd.agent.host=$DD_AGENT_HOST \
+     -Ddd.agent.port=$DD_TRACE_AGENT_PORT \
+     -jar /your/app.jar
+```
+
+{{% /tab %}}
+{{% tab "Python" %}}
+
+```python
```

Comment:
After adding the previous comment, you can say: "You can also use the Tracer configuration API" (or similar)